imports/ui/Task.js

```diff
@@ -1,14 +1,26 @@
 import { Meteor } from "meteor/meteor";
+import { ReactiveVar } from "meteor/reactive-var";
 import { Template } from "meteor/templating";
 
 import "./Task.html";
 
+Template.task.onCreated(function () {
+  this.isUpdateLoading = new ReactiveVar(false);
+});
+
 Template.task.events({
   "click .toggle-checked"(event) {
+    Template.instance().isUpdateLoading.set(true);
     Meteor.call("tasks.setIsChecked", this._id, !this.isChecked);
```

Comment:
This is callback function, so method call won't wait to end to call `Template.instance().isUpdateLoading.set(false);`

@AbegaM please change this call to `async/await` or callback type. Do not forget to check and log for errors if any returned from the server

Reply:
I have added a `try catch` function to catch and log errors that are coming from the backend server
 ```js
   try {
       Template.instance().isUpdateLoading.set(true);
       await Meteor.call("tasks.setIsChecked", this._id, !this.isChecked);
       Template.instance().isUpdateLoading.set(false);
     } catch (error) {
       console.log(error);
    }
 ```